pythonforandroid/bootstraps/sdl2/build/src/org/kivy/android/PythonActivity.java

```diff
@@ -178,6 +111,100 @@ public void run() {
         }
     }
 
+    private class UnpackFilesTask extends AsyncTask<String, Void, String> {
+        @Override
+        protected String doInBackground(String... params) {
+            File app_root_file = new File(params[0]);
+            Log.v(TAG, "Ready to unpack");
+            unpackData("private", app_root_file);
+            return null;
+        }
+
+        @Override
+        protected void onPostExecute(String result) {
+            // Figure out the directory where the game is. If the game was
+            // given to us via an intent, then we use the scheme-specific
+            // part of that intent to determine the file to launch. We
+            // also use the android.txt file to determine the orientation.
+            //
+            // Otherwise, we use the public data, if we have it, or the
+            // private data if we do not.
+            mActivity.finishLoad();
+
+            // finishLoad called setContentView with the SDL view, which
+            // removed the loading screen. However, we still need it to
+            // show until the app is ready to render, so pop it back up
+            // on top of the SDL view.
+            this.showLoadingScreen();
```

Comment:
Actually, I think this should be  mActivity.showLoadingScreen() - I don't know why it worked for me before, but it's crashing now.